pkg/applyinator/applyinator.go

```diff
@@ -73,7 +73,8 @@ type PeriodicInstructionOutput struct {
 
 type OneTimeInstruction struct {
 	CommonInstruction
-	SaveOutput bool `json:"saveOutput,omitempty"`
+	SaveOutput                        bool `json:"saveOutput,omitempty"`
+	ResetFailureCountOnServiceRestart bool `json:"resetFailuresOnServiceRestart,omitempty"`
```

Comment:
```suggestion
	ResetFailureCountOnStartup bool `json:"resetFailureCountOnStartup,omitempty"`
```
Name should match json, and I'd prefer `Startup` over `ServiceRestart` just due to the fact that it's shorter but has the same meaning.

Reply:
Updated, `r/r` pr is updated to use new json tag